### PR TITLE
added nanobox as a deployment option

### PR DESCRIPTION
**Pull request by sdomino**
Adding Nanobox as a development/deployment option for Hugo. Nanobox also provides a way to deploy applications to a number of hosts, but it itself is not a hosting platform.

### CODE REVIEW

File: content/hosting-and-deployment/deployment-with-nanobox.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+{{< /code >}}
+
+{{% note %}}
+If you already have a functioning Hugo app, which should now be configured, you can skip to [Deploying Hugo with Nanobox](#deploying-hugo-with-nanobox).
+{{% /note %}}
+
+### Installing Hugo
+
+Nanobox uses Docker to create instant, isoloated, development environments. Because of this, you'll need to make sure that during development you have Hugo available.
```
Comment (digitalcraftsman) [inline content/hosting-and-deployment/deployment-with-nanobox.md]: "isoloated" looks like a typo ;)

File: content/hosting-and-deployment/deployment-with-nanobox.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+```
+# copy the config.toml that comes with the theme
+cp ./themes/nanobox-hugo-theme/config.toml config.toml
+
+# or, add it to your existing config.toml
+theme = "nanobox-hugo-theme"
+```
+
+{{% note %}}
+It is not intended that you use the `nanobox-hugo-theme` as your actual them. It's simply a theme to start with and should be replaced.
```
Comment (digitalcraftsman) [inline content/hosting-and-deployment/deployment-with-nanobox.md]: them -> theme

**Comment by sdomino** [comment]
@digitalcraftsman, thanks for finding those and pointing them out. I've committed the corrections.

**Comment by digitalcraftsman** [comment]
Well, thanks for your contribution :+1:

**Pull request merged**